common/test/test_migrations/utils.py

```diff
@@ -0,0 +1,59 @@
+"""
+General class for Test migrations.Based off an implementation provided at
+https://www.caktusgroup.com/blog/2016/02/02/writing-unit-tests-django-migrations/
+"""
+# pylint: disable=redefined-outer-name
+
+from django.test import TransactionTestCase
+from django.db.migrations.executor import MigrationExecutor
+from django.db import connection
+
+
+class TestMigrations(TransactionTestCase):
+    """ Base class for testing migrations. """
+    migrate_from = None
+    migrate_to = None
+    app = None
+
+    def setUp(self):
+        super(TestMigrations, self).setUp()
+        self.check_data()
+
+    def execute_migration(self, previous, next):
+        """
+        Execute migration from state to another.
+        """
+        # Reverse to the original migration
+        self.executor.migrate(previous)
+
+        self.setUpBeforeMigration()
+
+        # Run the migration to test
+        self.executor.migrate(next)
+
+    def check_data(self):
```

Comment:
I'd just put this in `setUp`. I'm not sure `check_data` is the most descriptive term for this anyway.